lib/libutils/ext/trace.c

```diff
@@ -75,6 +75,8 @@ void trace_printf(const char *function, int line, int level, bool level_ok,
 	size_t boffs = 0;
 	int res;
 
+	memset(buf, 0, sizeof(buf));
```

Comment:
Yes, I think you're right. I first added this to ensure a NULL character always exists, but since I still always add it this further down, the memset doesn't makes sense. I'll remove it and add your RB-tags.